creator/data_templates/models.py

```diff
@@ -0,0 +1,79 @@
+import uuid
+from django.db import models
+from django.contrib.auth import get_user_model
+
+from creator.fields import kf_id_generator
+from creator.organizations.models import Organization
+
+User = get_user_model()
+
+
+def template_id():
+    return kf_id_generator("DT")
+
+
+class DataTemplate(models.Model):
+    """
+    A data submission template with versions
+
+    The Template model represents the essence of the data template just like
+    File does with Version, and it provides a container for the versions
+    of the template
+    """
+
+    class Meta:
+        permissions = [
+            ("list_all_datatemplate", "Show all data_template"),
+        ]
+
+    id = models.CharField(
+        max_length=11,
+        primary_key=True,
+        default=template_id,
+        help_text="Human friendly ID assigned to the data template",
+    )
+    uuid = models.UUIDField(
+        default=uuid.uuid4, help_text="UUID used internally"
+    )
+    created_at = models.DateTimeField(
+        auto_now=True,
+        null=True,
+        help_text="Time when the data template was created",
+    )
+    modified_at = models.DateTimeField(
+        auto_now_add=True,
+        null=True,
+        help_text="Time when the data template was modified",
+    )
```

Comment:
These are mixed up, `auto_now` updates every time the object is saved:
```suggestion
created_at = models.DateTimeField(
        auto_now_add=True,
        null=True,
        help_text="Time when the data template was created",
    )
    modified_at = models.DateTimeField(
        auto_now=True,
        null=True,
        help_text="Time when the data template was modified",
    )
```

Reply:
Arrgh oops, thanks!